backend/src/server/routes/sanitizedSchemas.ts

```diff
@@ -32,7 +32,7 @@ export const DefaultResponseErrorsSchema = {
   400: z.object({
     requestId: z.string(),
     statusCode: z.literal(400),
-    message: z.string(),
```

Comment:
1. Revert this back to `z.string()`. 
2. Set the `401` message to `z.string()` instead of any, since we now know 401 message will be a string. 
3. Create a new item in the object for 422 errors so we can get proper API documentation for it.

```ts
  422: z.object({
    requestId: z.string(),
    statusCode: z.literal(422),
    message: z.any(),
    error: z.string()
  }),

Reply:
ah good point!